packages/@aws-cdk/aws-eks/lib/cluster-resource-handler/cluster.ts

```diff
@@ -136,23 +136,32 @@ export class ClusterResourceHandler extends ResourceHandler {
       return this.updateClusterVersion(this.newProps.version);
     }
 
-    if (updates.updateLogging || updates.updateAccess) {
+    if (updates.updateLogging && updates.updateAccess) {
```

Comment:
the EKS API does not allow both to be updated at the same time, but if we make 2 separate API calls, do we need to set this restriction on the custom resource?

Reply:
I am not sure if this is a good idea to have a custom resource making multiple API calls. Would love to hear comments from @iliapolo or any maintainers before we move this PR further.

Reply:
We have precedents for this already in other custom resources. Some examples involving multiple write calls:

* In S3, for automatic deletion of objects, we call `deleteObjects` multiple times.
* In IAM, for the OIDC provider, we call many IAM methods, such as `updateOpenIDConnectProviderThumbprint`, `addClientIDToOpenIDConnectProvider` and `removeClientIDFromOpenIDConnectProvider`.
* In Logs, for log retention, we call `createLogGroup`, `deleteRetentionPolicy` and `putRetentionPolicy`. 

And, if we also consider read calls, pretty much every custom resource makes a combination of read and write calls.

Reply:
Given that, I agree we should making multiple API calls in this custom resource but I probably don't have capacity for that in the next few weeks. I will make this PR as is for immediate fix and leaving the multiple API calls implementation for another PR if anyone is interested.